packages/xapi-rrd/xapi-rrd.1.0.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "xen-api@lists.xen.org"
+authors: ["Dave Scott" "Jon Ludlam" "John Else"]
+homepage: "https://github.com/xapi-project/xcp-rrd"
+bug-reports: "https://github.com/xapi-project/xcp-rrd/issues"
+dev-repo: "https://github.com/xapi-project/xcp-rrd.git"
+tags: [
+  "org:xapi-project"
+]
+build: [
+  [make]
+]
+build-test: [
+  [make "test"]
+]
+install: [
+  [make "PREFIX=%{prefix}%" "install"]
+]
+remove: [
+  [make "PREFIX=%{prefix}%" "uninstall"]
+]
+depends: [
+  "ocamlfind" {build}
+  "oasis"     {build}
+  "base-bigarray"
+  "base-unix"
+  "rpc" {>= "1.9.51"}
+  "uuidm"
+  "ounit" {test}
```

Comment:
That's strange. Do you have a specific branch with something that breaks? Maybe there is some depopts failure somewhere. 

I really can't wait to move to jbuilder and get rid of them. 

I am preparing a PR with xapi-rrdd and xapi-networkd, I can add a fix there if we figure out the issue